editors/emacs/Portfile

```diff
@@ -142,9 +143,20 @@ if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
 
     livecheck.type none
 
+    # Emacs 30 removes libjansson in favour of native JSON support.
```

Comment:
I doubt the additions here follow the modeline with respect to alignment (i.e., multiple of four spaces).

Reply:
Didn't realise that applied after the first non-whitespace character on the line as there are prior lines in this very file which do not follow the convention. Perhaps an autoformatter is required? I corrected some other ones (not mine) as well as my ones, but there are others in this file which would be too tedious to do manually.